src/components/api-request.js

```diff
@@ -1050,7 +1050,9 @@ export default class ApiRequest extends LitElement {
       let respJson;
       let respText;
       tryBtnEl.disabled = true;
+      const fetchStart = new Date();
       fetchResponse = await fetch(fetchRequestObject);
+      this.elapsedMs = new Date() - fetchStart;
```

Comment:
To make sure this is reactive and shows up correctly, can we add the new property to:
* the constructor
* `clearResponseData`
* `get properties()` - (should be type of Number)

Reply:
Done -- hope I did this right :)